subsys/nrf_rpc/crash_gen/common/crash_gen_rpc_ids.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef CRASH_GEN_RPC_IDS_H_
+#define CRASH_GEN_RPC_IDS_H_
+
+/** @brief Command IDs accepted by the crash generator over RPC server.
+ */
+enum crash_gen_rpc_cmd_server {
```

Comment:
Just a thought, do you think it's worth creating a separate group for these 3 commands (which could also be a single command with an enum argument)? The reason I'm asking is that adding a new group takes a bigger effort because you need to add a new library within NCS and also each group means more data that need to be exchanged to establish the RPC channels.

I can see that Łukasz added the remote shell command to `dev_info` group so maybe some sort of `test_utils` groups could be used for other ad-hoc test-only features?